app/os_android.go

```diff
@@ -951,3 +959,59 @@ func Java_org_gioui_Gio_scheduleMainFuncs(env *C.JNIEnv, cls C.jclass) {
 }
 
 func (_ ViewEvent) ImplementsEvent() {}
+
+var foregroundService struct {
+	once   sync.Once
+	intent C.jobject
+	mu     sync.Mutex
+	count  int
+}
+
+// StartForeground starts the foreground service
+func StartForeground(title, text string) (err error) {
```

Comment:
It's not great to have `StartForeground` exported. Can you move the implementation to package foreground?

Reply:
I cannot move StartForeground out of package app without exporting methods used from app.